README.md

```diff
@@ -456,6 +471,17 @@ Pushes the branch to `origin` and then creates a Bitbucket pull request for the
     - Create an application password within your [account settings](https://bitbucket.domain.com/plugins/servlet/access-tokens/manage).
     - We need the scope: Repositories -> Read
 
+### `all_repos.push.bitbucket_server_pull_request_bearer_auth`
+
+Pushes the branch to `origin` and then creates a Bitbucket pull request for the branch using bearer auth scheme.
+
+#### Required `push_settings`
+
+- `base_url`: the Bitbucket server URL (eg `bitbucket.domain.com`)
+- `token`: the personal access token which the user will log in
+    - Create a personal access token within your [account settings](https://bitbucket.domain.com/plugins/servlet/access-tokens/manage).
+    - We need the scope: Repositories -> Read
```

Comment:
presumably you'd need higher than `read` to push ?

Reply:
No, Read scope is good enough as I have been using it for a long time.

Reply:
how do you make a push with a readonly token?

Reply:
Updated the patch.